test/e2e/network/networking_perf.go

```diff
@@ -19,6 +19,10 @@ package network
 // Tests network performance using iperf or other containers.
 import (
 	"fmt"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
```

Comment:
rearrange the imports please, usually standards first and together

Reply:
👍